frontend/tests/search.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from "@playwright/test";
+import { navigateTo } from "./common";
+
+
+test("Search", async ({ page }) => {
+    await navigateTo("/", page);
+    await page.waitForSelector("nav");
+    const searchField = page.locator("_react=SearchField").locator("input");
```

Comment:
Same as a previous comment: I think it would be better to just select "an input where the placeholder is 'search'" or sth like that. At least my very little knowledge about UI tests tells me to code them as close to what a user would do.